### PR TITLE
 Display associations in a list:: edit css code

### PR DESCRIPTION
**Pull request by medou-boushab**

### CODE REVIEW

File: app/views/common/catalog-view.html [hunk 1]
```diff
@@ -16,11 +16,11 @@
         </form>
         <div class="container-flex m-l-md btn-group">
             <button class="btn btn-default"  ng-click="toggleDisplay()" tooltip-append-to-body="true" uib-tooltip="{{showPSAWorkflowsOnly?'Show services as icons':'Show workflows as icons'|translate}}" ng-model="toggleListBox.value"
-                    uib-btn-radio="false">
+                    uib-btn-radio="false" ng-class="{'btn-green':!toggleListBox.value}">
```
Comment (amouhoub) [inline app/views/common/catalog-view.html:19]: @medou-boushab  why no use `btn btn-primary` ? it's already in inspinia and we use it everywhere ?

File: app/views/common/catalog-view.html [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
                 <i class="fa fa-th-large"></i>
             </button>
             <button class="btn btn-default" ng-click="toggleDisplay()" ng-model="toggleListBox.value" tooltip-append-to-body="true" uib-tooltip="{{showPSAWorkflowsOnly?'Show services as a list':'Show workflows as a list'|translate}}"
-                    uib-btn-radio="true">
+                    uib-btn-radio="true" ng-class="{'btn-green':toggleListBox.value}">
```
Comment (amouhoub) [inline app/views/common/catalog-view.html:23]: same comment as above

File: app/styles/dashboard-style.css [hunk 3]
```diff
@@ -872,6 +900,13 @@ md-virtual-repeat-container:-webkit-scrollbar {
     float: right;
 }
 
+.btn-green,
+.btn-green:hover {
```
Comment (amouhoub) [inline app/styles/dashboard-style.css:904]: why force the hover behaviour to be the same as the normal state ? you're removing the hover effect and I don't see why...

File: app/views/common/catalog-view.html (hunk 1 above)
Comment (medou-boushab) [reply app/views/common/catalog-view.html:19]: you're right, my bad

**Pull request merged**